keyserver/coniksserver/main.go

```diff
@@ -0,0 +1,146 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"path"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+	"github.com/coniks-sys/coniks-go/crypto/vrf"
+	"github.com/coniks-sys/coniks-go/keyserver"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+type boolStringFlag struct {
+	set   bool
+	value string
+}
+
+func (sf *boolStringFlag) Set(x string) error {
+	sf.value = x
+	sf.set = true
+	return nil
+}
+
+func (sf *boolStringFlag) String() string {
+	return sf.value
+}
+
+func (sf *boolStringFlag) IsBoolFlag() bool {
+	return true
+}
+
+func main() {
+	var genkey boolStringFlag
+	flag.Var(&genkey, "genkey", "Generate signing/VRF key")
```

Comment:
Hmm, `boolStringFlag` doesn't seem to be working out the way you wanted it to ... at least, the value always seems to be true, and not a string.

Should probably just make change it to,
```
genkey := flag.String("genkey", "sign,vrf", "Generate signing/VRF key")
```